app/assets/stylesheets/pin-show.css.scss

```diff
@@ -0,0 +1,117 @@
+$primary-background-color: rgba(237, 242, 242, 0.6);
+
+
+.container-fluid{
+  background-color: $primary-background-color;
+}
+
+.row{
```

Comment:
Same thing, you're messing up with bootstrap + you're being too specific because now all .row in the app will have this new rule